### ISSUE TITLE
1.07-r1 is on main Play Store branch

### ISSUE THREAD

**Issue by JJRcop** [PhilippC/keepass2android#889]
Version 1.07-r1 has been on the main Play Store branch since Apr 9, 2019.

This may have been deliberate, but issues like #799 mean version 1.07-r1 is not what I consider stable for the main branch.

I have had to manually install 1.06 from github since updating to the latest on the store updates to 1.07-r1, which is unusable for me because of #799.

What I expect is for the beta channel to have 1.07-r1 and the main channel to have 1.06.

This likely conflicts with #848

**Comment by JohnLGalt** [comment]
Agreed - 1.07 should be relegated to ẞeta status again, and removed from stable status.

**Comment by JohnLGalt** [comment]
And now Git is showing a release of 1.07 r1 as of 2 days ago.

WTF?

https://github.com/PhilippC/keepass2android/releases/tag/1.07

**Comment by JJRcop** [comment]
Looks like #799 is fixed in 1.07b-r0.

This issue was written under the assumption that these versions are pre-release versions or release candidates, but if they are not and are intended to be the main release versions, then this issue does not serve any purpose and can be closed.

@PhilippC, do you have any comment on this, are the 1.07-r1 and 1.07b-r0 versions release versions, or release candidates/pre-release?
